core/tests/protractor_utils/PreferencesPage.js

```diff
@@ -34,6 +34,13 @@ var PreferencesPage = function() {
   var subscriptions = element.all(by.css('.protractor-test-subscription-name'));
   var systemLanguageSelector = element.all(
     by.css('.protractor-test-system-language-selector')).first();
+  var bioLink = element(by.css('.protractor-test-preferences-editBio'));
+
+  this.addBio = function(bio) {
```

Comment:
Maybe setUserBio? (Add sounds like we are appending. I hope that's not what you want to achieve here (Append), right?)

Reply:
The task wad "Add a bio to your profile ", that's why i called it this way. But it's ok, I have changed the name to setUserBio